intel_extension_for_transformers/neural_chat/examples/serving/triton_inference_sever/hpu/README.md

```diff
@@ -0,0 +1,102 @@
+# Serving NeuralChat Text Generation with Triton Inference Server on HPU
+
+Nvidia Triton Inference Server is a widely adopted inference serving software. We also support serving and deploying NeuralChat models with Triton Inference Server.
+
+## Prepare serving scripts
+
+```bash
+cd <path to intel_extension_for_transformers>/neural_chat/examples/serving/triton_inference_server
+mkdir -p models/text_generation/1/
+cp ../../../serving/triton/text_generation/model.py models/text_generation/1/model.py
+cp ../../../serving/triton/text_generation/client.py models/text_generation/1/client.py
+cp ../../../serving/triton/text_generation/config.pbtxt models/text_generation/config_hpu.pbtxt
```

Comment:
should be ../../../serving/triton/text_generation/config_hpu.pbtxt models/text_generation/config.pbtxt

Reply:
Modified, thanks.